scanners/pageload.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from phantomas import Phantomas, PhantomasError
+
+command = Phantomas.CMD
+init = None
+
+def scan(domain, options):
+    logging.debug("[%s][pageload]" % domain)
+
+    if not (domain.startswith('http://') or
+            domain.startswith('https://')):
+        domain = 'http://' + domain
```

Comment:
> However, we could easily use the inspect data if it is available (as you do in the linked TLS scanner code), and otherwise fall back onto this existing logic.

Exactly.